itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/universal/methods/AbstractQuarkusTest.java

```diff
@@ -110,29 +107,17 @@ public static void checkProblemsView() {
 
 	}
 
-	public static void checkUrlContent(String should_be) {
-		String urlContent = "";
-
-		try {
-			urlContent = getUrlContent("http://localhost:8080/hello");
-		} catch (IOException e) {
-			QuarkusCorePlugin.logException("Wrong URL!", e);
-		}
-
-		assertEquals("Should be <" + should_be + "> , but is <" + urlContent + ">", should_be, urlContent);
-	}
-
-	public static String getUrlContent(String readedURL) throws IOException {
-
-		URL localhost = new URL(readedURL);
-		String inputLine = "";
-		try (BufferedReader in = new BufferedReader(new InputStreamReader(localhost.openStream()));) {
-			inputLine = in.readLine();
-			in.close();
-		} catch (IOException e) {
-			QuarkusCorePlugin.logException("Can`t read from url!", e);
-		}
-		return inputLine;
+	public static void createNewFile(String projectName, String fileName, String filePath) {
+		new WorkbenchShell().setFocus();
+		ProjectExplorer pe = new ProjectExplorer();
+		pe.selectProjects(projectName);
+		pe.getProject(projectName).getProjectItem(filePath).select();
+		new ContextMenuItem(TextLabels.NEW_CONTEXT_ITEM, "Other...").select();
+		new LabeledText("Wizards:").setText("File");
+		new NextButton().click();
+		new LabeledText("File name:").setText(fileName);
```

Comment:
and also this: https://github.com/eclipse/reddeer/blob/37f27affd709a9622d84e8d29d047dc78a0de1b4/plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/ui/dialogs/WizardNewFileCreationPage.java

Reply:
reworked